spec/features/basic_search_spec.rb

```diff
@@ -0,0 +1,110 @@
+# spec/register_down_spec.rb
+
+require 'page'
+
+RSpec.feature "Verify query search results" do
+
+  subject { Page.new }
+
+  background do
+    subject.visit 'https://google.com'
+    subject.enter_keyword keyword
+  end
+
+  feature "Use one-word keyword" do
+
+    context "Ignore case" do
+
+      context "Downcase input" do
+
+        given(:keyword) {'monster'}
+
+        scenario "Title contains inputted keyword Upcase" do
+          expect(subject.first_result_title).to include(keyword.capitalize)
+        end
+
+        scenario "URL contains inputted keyword Downcase" do
+          expect(subject.first_result_url).to include(keyword.downcase)
+        end
+
+        scenario "Description contains inputted keyword Upcase" do
+          expect(subject.first_result_description).to include(keyword.capitalize)
+        end
+
+      end # context downcase
+
```

Comment:
You like empty lines so much. I recommend to use [ruby-style-guide](https://github.com/bbatsov/ruby-style-guide).